contents/handbook/company/team/joe-martin.md

```diff
@@ -20,25 +20,23 @@ I live just outside of London. I'm passionate about punk music, videogame preser
 - I enjoy helping others. You can ask me anything. There are no stupid questions.  
 
 ### How to work with me
-- I default to listening. You may need to ask for my thoughts if you want them.
+- I can be over-communicative and need to be challenged on this. Tell me if you want the short version.
```

Comment:
it's epic to see diffs like these 😍

Reply:
Yeah, a variety of things recently have made me realise I talk (and type) _way_ too much once I'm in a position where I'm comfortable with the overall rapport and familiar with the people I'm engaging with. Its something I need to work on.

Reply:
Lower your issue : pull request ratio!